staging/src/k8s.io/apiserver/pkg/server/resourceconfig/helpers.go

```diff
@@ -149,7 +148,7 @@ func ParseGroups(resourceConfig utilflag.ConfigurationMap) ([]string, error) {
 			continue
 		}
 		tokens := strings.Split(key, "/")
-		if len(tokens) != 2 && len(tokens) != 3 {
+		if len(tokens) < 2 {
```

Comment:
!= 2 ?

Reply:
I reused the ParseGroups for kube-apiserver

Reply:
also it only cares about group. No matter it is group/version or group/version/resource. 

APIEnablementOptions.Validate() already verifies ==2 or ==3 before verifying groups.